init_db.py

```
@@ -0,0 +1,26 @@
+from src.api.app import app, db
+from src.api.models import User
+from werkzeug.security import generate_password_hash
+
+def init_db():
+    with app.app_context():
+        # Create tables
+        db.create_all()
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)